package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Charles Lowell <cowboyd@frontside.io>",
   "license": "MIT",
   "private": false,
-  "types": "types/index.d.ts",
+  "types": "index.d.ts",
```

Comment:
Isn't this change in master already? Not sure why we are getting a diff here.

Reply:
I tried to rebase and include the fix and I think I mucked it up 😕